http/backup_service.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/influxdata/httprouter"
 	"github.com/influxdata/influxdb/v2"
+
```

Comment:
This blank line also surprises me (can we delete the commented-out line below instead?)